tests/files/crd/networking.kubeslice.io_slices.yaml

```diff
@@ -6,20 +6,20 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.7.0
   creationTimestamp: null
-  name: slice.networking.kubeslice.io
+  name: slices.networking.kubeslice.io
```

Comment:
Don think the api name change and change of file name is required 
PTAL: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition